library/analytics/src/main/java/org/cru/godtools/analytics/model/AnalyticsScreenEvent.kt

```diff
@@ -31,6 +31,7 @@ open class AnalyticsScreenEvent @JvmOverloads constructor(val screen: String, lo
 
         /* Adobe Site Sections */
         const val ADOBE_SITE_SECTION_TOOLS = "tools"
+        const val SCREEN_GLOBAL_DASHBOARD = "Global Dashboard"
```

Comment:
this isn't an adobe site section, it shouldn't be under the `Adobe Site Sections` constants